staging/src/k8s.io/apimachinery/pkg/util/managedfields/fieldmanager_test.go

```diff
@@ -883,100 +874,6 @@ func TestUpdateViaSubresources(t *testing.T) {
 	}
 }
 
-// Ensures that a no-op Apply does not mutate managed fields
```

Comment:
I don't see how this is different from [`TestNoSemanticUpdateApplySameResourceVersion`](https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/apply/apply_test.go#L302-L384)